### PR TITLE
[openthermgateway] Added support for firmware version 5.0

### PR DESCRIPTION
**Pull request by ArjenKorevaar**
See #10038

### CODE REVIEW

**Comment by ArjenKorevaar** [comment]
Added support for:
- C2 and H2 commands to control a second central heating circuit 
- TS command to select temperature sensor type

File: bundles/org.openhab.binding.openthermgateway/src/main/resources/OH-INF/thing/channels.xml [hunk 1]
```diff
@@ -60,6 +60,30 @@
 		<state readOnly="false" min="0" max="100" step="0.1" pattern="%.1f %unit%"/>
 	</channel-type>
 
+	<channel-type id="controlsetpoint2">
+		<item-type>Number:Temperature</item-type>
+		<label>Control Setpoint2 </label>
```
Comment (Hilbrand) [inline bundles/org.openhab.binding.openthermgateway/src/main/resources/OH-INF/thing/channels.xml:65]:
```suggestion
		<label>Control Setpoint 2</label>
```

File: bundles/org.openhab.binding.openthermgateway/src/main/resources/OH-INF/thing/channels.xml [hunk 2]
```diff
@@ -133,6 +157,28 @@
 		<item-type>Switch</item-type>
 		<label>Central Heating Overridden</label>
 		<description>Central heating enabled overridden at OTGW</description>
+		<state readOnly="false"/>
+	</channel-type>
+
+	<channel-type id="ch2_enable">
+		<item-type>Switch</item-type>
+		<label>Central Heating Enabled</label>
```
Comment (Hilbrand) [inline bundles/org.openhab.binding.openthermgateway/src/main/resources/OH-INF/thing/channels.xml:165]: All `ch2_*` channel-type definitions miss `2` in the label and description.
```suggestion
		<label>Central Heating 2 Enabled</label>
```

**Review by Hilbrand — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.openthermgateway/src/main/resources/OH-INF/thing/channels.xml (hunk 2 above)
Comment (ArjenKorevaar) [reply bundles/org.openhab.binding.openthermgateway/src/main/resources/OH-INF/thing/channels.xml]: Added 2 to the labels and descriptions

File: bundles/org.openhab.binding.openthermgateway/src/main/resources/OH-INF/thing/channels.xml (hunk 1 above)
Comment (ArjenKorevaar) [reply bundles/org.openhab.binding.openthermgateway/src/main/resources/OH-INF/thing/channels.xml]: Fixed typo

**Review by Hilbrand — APPROVED** [review]
LGTM

**Pull request merged**